### PR TITLE
Classic Tutorial

### PR DESCRIPTION
**Pull request by nids2001**
Ports BIOPSE classic tutorial ([http://scirundocwiki.sci.utah.edu/SCIRunDocs/index.php/CIBC:Documentation:SCIRun:Tutorial:BioPSE:Chapter_1](http://scirundocwiki.sci.utah.edu/SCIRunDocs/index.php/CIBC:Documentation:SCIRun:Tutorial:BioPSE:Chapter_1)) to markdown as mentioned in #1156

### CODE REVIEW

**Review by dcwhite — DISMISSED** [review]

File: docs/user_doc/ClassicTutorial.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+
+## Saving and reloading networks
+
+Now that a three-module network has been created, save it to disk. The .srn5 file can easily be reloaded in a future SCIRun session.
+
+### Saving a SCIRun network:
+
+1. Click on the File menu (at the top of the Network Editor window) and select "Save As."
+2. When the file browser appears, follow the prompt to choose a location and filename for the network. Many example networks are stored in the ExampleNets directory, which is distributed with the binaries, or in SCIRun/src/ExampleNets. The network can be stored in any location with write access.  
+-TODO: update show-torso-mesh.srn to SR5 and add to example nets
```
Comment (jessdtate) [inline docs/user_doc/ClassicTutorial.md:251]: @nids2001 did you update this example network?

**Review by jessdtate — CHANGES_REQUESTED** [review]

File: docs/user_doc/ClassicTutorial.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+See [Appendix 1](#appendix-1) for a description of various types of geometric meshes, data values, and mappings SCIRun supports.
+
+
+## Hooking Modules Together
+
+Now add a second module to the network. This module is used to visualize various Field types. Then connect the two modules in the canvas so data can flow between them.
+
+1. Create a **ShowField** module using the DataIO section in the module selector (use the same menus used to create the ReadField module).
+2. Position the mouse pointer over the yellow output port on the **ReadField**. Press and hold the right mouse button. The name of the port and lines indicating possible data pipe connections will appear.
```
Comment (jessdtate) [inline docs/user_doc/ClassicTutorial.md:108]: left mouse button.

File: docs/user_doc/ClassicTutorial.md [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+3. Continue to hold the right middle button and drag the mouse toward the first yellow **ShowField** input port.
```
Comment (jessdtate) [inline docs/user_doc/ClassicTutorial.md:109]: left mouse buttion

File: docs/user_doc/ClassicTutorial.md [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+4. The line turns gray, showing the desired connection has been selected. See ({numref}`fig-pipe-selection`).
```
Comment (jessdtate) [inline docs/user_doc/ClassicTutorial.md:110]: turns red

File: docs/user_doc/ClassicTutorial.md [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+
+:::{figure-md} fig-connected-dataflow
+
+![Connected Dataflow Pipe](ClassicTutorial_figures/connecteddataflow.png)
+
+Connected Dataflow Pipe
+:::
+
+
+## Dynamic Commpilation
```
Comment (jessdtate) [inline docs/user_doc/ClassicTutorial.md:129]: this isn't supported anymore, so this section should be removed

File: docs/user_doc/ClassicTutorial.md [hunk 6]
```diff
@@ -0,0 +1,307 @@
...
+Dynamic Compilation causes a delay when the module executes the first time. For more details, refer to [Dynamic Compilation](http://www.sci.utah.edu/publications/mcole01/dyn.pdf).
+
+In addition, the module turns light yellow while it compiles, and compilation output goes to the module's log as a remark. Note that SGI computers can take over 30 seconds to perform dynamic compilation. When compilation is done, the modules color returns to a light gray.
+
+
+### Setting the ShowField User INTERFACE_MODULES_
+
+The **ShowField** module has options for changing the visual representations of a Field's geometry. To illustrate the module's functionality, change **ShowField** parameters using its GUI. Specifically, change the color of the nodes to blue spheres.
+
+1. Select the UI button on the **ShowField** module.
```
Comment (jessdtate) [inline docs/user_doc/ClassicTutorial.md:142]: reference fig-module-info

File: docs/user_doc/ClassicTutorial.md [hunk 7]
```diff
@@ -0,0 +1,307 @@
...
+
+:::{figure-md} fig-vs-window
+
+![ViewScene window showing the utahtorso-lowres-voltage data](ClassicTutorial_figures/vswindow.png)
+
+ViewScene window showing the utahtorso-lowres-voltage data
+:::
+
+
+## Mouse Controls
```
Comment (jessdtate) [inline docs/user_doc/ClassicTutorial.md:184]: these are the old controls

**Review by jessdtate — CHANGES_REQUESTED** [review]

**Review by jessdtate — APPROVED** [review]

**Review by dcwhite — APPROVED** [review]

**Pull request merged**